src/components/Navigation/Navigation.js

```diff
@@ -1,11 +1,20 @@
 import React from 'react';
+import { FaUserAlt } from 'react-icons/fa';
 
 import { Link } from 'react-router-dom';
 
 const Navigation = () => (
-  <nav>
-    <Link to="/">Books</Link>
-    <Link to="/categories">Categories</Link>
+  <nav className="bg-white w-full flex justify-between items-center px-14 py-6 shadow-md">
+    <div className="flex items-center space-x-7">
+      <h2 className="text-blue-600  font-bold text-3xl">
+        Welcome to BookStore
```

Comment:
- [ ] As the requirements said you need to make your page exactly like the design so here use `BOOKSTORE CMS`